<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationClass</class>
 <widget class="QMainWindow" name="VisualizationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Visualization.qrc">
    <normaloff>:/Visualization/images/main_icon.png</normaloff>:/Visualization/images/main_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuParallel_Coordinates">
     <property name="title">
      <string>Parallel Coordinates</string>
     </property>
     <addaction name="actionNone"/>
     <addaction name="actionBoth"/>
     <addaction name="actionTime"/>
     <addaction name="actionManufacturer"/>
    </widget>
    <addaction name="menuParallel_Coordinates"/>
    <addaction name="separator"/>
    <addaction name="actionTable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Visualization.qrc">
     <normaloff>:/Visualization/images/file_open.png</normaloff>:/Visualization/images/file_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Visualization.qrc">
     <normaloff>:/Visualization/images/file_new.png</normaloff>:/Visualization/images/file_new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Visualization.qrc">
     <normaloff>:/Visualization/images/file_save.png</normaloff>:/Visualization/images/file_save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionManufacturer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manufacturer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actionBoth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Both</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Visualization.qrc"/>
 </resources>
 <connections/>
</ui>
